### PR TITLE
update landing page

### PR DESCRIPTION
**Pull request by flossypurse**
**PREVIEW**: https://deploy-preview-994--mystifying-fermi-1bc096.netlify.app/

### CODE REVIEW

**Review by djmagee — APPROVED** [review]
Also add links to first app and Hello World to the docs/learning-paths/index.md page?

File: docs/learning-paths/index.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ id: index
 title: Learning
 ---
 
-- Run your first app
-- Hello World from scratch
+- [Run your first app](/docs/learing-paths/run-your-first-app)
```
Comment (flossypurse) [inline docs/learning-paths/index.md:6]:
```suggestion
- [Run your first app](/docs/learning-paths/run-your-first-app)
```

**Comment by sw-yx** [comment]
RIP all the temporal learning video content :)

**Comment by flossypurse** [comment]
> RIP all the temporal learning video content :)

Not deleting any video content - non-docs links have simply been redistributed to a more appropriate location.

If you have genuine concerns here - then let's take a look.

**Pull request merged**